packages/core/tooling/index.ts

```diff
@@ -207,43 +213,55 @@ export function guessIndentString(str: string | undefined): string {
 	}
 }
 
-export function guessQuoteStyle(ast: TsEstree.Node): 'single' | 'double' | undefined {
```

Comment:
Any reason to remove `guessQuoteStyle` ?

Reply:
The arg was just moved to a different place. Fixed.